### DISCUSSION TITLE
Installation question

### DISCUSSION THREAD

**Discussion by itrn57** [j-kaltes/Juggluco#6] [Q&A] [answered]
Hi, sorry for a silly question.
When upgrading from v. 4.3.4 to the latest v. 4.8.2 would it be better to uninstall v. 4.3.4 first or it does not matter ?
Cheers

**Comment by j-kaltes** [comment, marked as answer]
You can better update, because otherwise you loose data. It will certainly give no problems.

**Reply by itrn57** [reply]
Makes sense, thank you. I assume same applies to upgrading the watch app, right?

**Reply by j-kaltes** [reply]
Yes, the only difference is that you don't lose your data when you reinstall the watch app. (If you want to reset the settings, it isn't so a bad idea, but then you also have to remove the old mirror connection in the phone version.)

**Comment by itrn57** [comment]
What version would you suggest for Galaxy Watch 5 - "Arm32 only" or "Arm/Arm64/x86/x86_64" ? apparently it supports bot 32 and 64 bits apps. 
Thanks.

**Comment by j-kaltes** [comment]
Take Arm32, that uses less RAM. 
On Galaxy Watch 4, you can only run Arm32 apps and Watch 5 differs little from Watch4. Nowhere they mention Arm64.

**Comment by itrn57** [comment]
Installed Arm32 on Watch5 following instructions. All works fine, thank you. 
Is it possible to keep floating glucose enabled but disable a notification about it? It seems like notification appears on every sensor read.

**Reply by j-kaltes** [reply]
You can disable the glucose notification of the phone version by unchecking Left menu->Settings->"Glucose in android statusbar".
